transaction/core/src/validation/validate.rs

```diff
@@ -194,6 +196,7 @@ fn validate_outputs_public_keys_are_unique(tx: &Tx) -> TransactionValidationResu
 }
 
 /// All outputs have a memo (old-style TxOuts are rejected)
+#[allow(unused)]
```

Comment:
Why is this required?

Reply:
it's required because it's not used except in tests, so during `cargo build` we get an unused function warning.

we intend to bring this back in resolution to #907 , so i figured it makes sense to just keep the code here and keep it under test, and then start using it in the main `validate()` function when we're ready to do that

Reply:
Why not `#[cfg(test)]`?

Reply:
that should be fine too